app/scripts/modules/core/loadBalancer/LoadBalancers.tsx

```diff
@@ -153,6 +153,8 @@ export class LoadBalancers extends React.Component<IProps, IState> {
       </div>
     );
 
+    const tagCleared = () => this.updateLoadBalancerGroups();
```

Comment:
whoops, yeah, I didn't need that. I thought I did because originally I had...it doesn't matter, I can move this into a private method to get around weird `this` binding issues.